### PR TITLE
Fix issues where two keys always had the same binding

### LINKED ISSUE #5: Compiling firmware causes duplicate keys
I've noticed an issue when trying to compile the vial firmware to increase the layer count.

The two keys marked "3,7" in this image seem to have their functionality tied together.
image

The issue isn't present in buran_kle.json, but somehow still impacts compiled firmware

### PR DESCRIPTION
**Pull request by Ethanvdw**
Keys 3,7 and 3,8 (lower right corner) were both set to 3,7.

This meant that in vial they would always be set to the same function.

Fixes #5

### CODE REVIEW

**Comment by Ethanvdw** [comment]
Also increased layer count to 8 and removed an inaccessible layer

File: firmware/vial/buran/config.h [hunk 1]
```diff
@@ -13,13 +13,16 @@
 
 /* --- Matrix definition --- */
 #define MATRIX_ROWS 4 
-#define MATRIX_COLS 11
+#define MATRIX_COLS 12
 
 /* --- Matrix assignment --- */ 
 #define MATRIX_COL_PINS \
-    { GP7, GP8, GP9, GP11, GP10, GP0, GP17, GP18, GP19, GP20, GP21 }
+    { GP7, GP8, GP9, GP11, GP10, GP0, GP17, GP18, GP19, GP20, GP21, GP16}
```
Comment (ScatteredDrifter) [inline firmware/vial/buran/config.h:20]: oh well I think I've fixed that locally once, yet never pushed it. Sorry and thank you for fixing that!

File: firmware/vial/buran/keymaps/vial/config.h [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 #define VIAL_KEYBOARD_UID {0x3F, 0x25, 0xD9, 0xFC, 0xA2, 0x98, 0x8F, 0x98}
 
+#define DYNAMIC_KEYMAP_LAYER_COUNT 8
 
 #define VIAL_UNLOCK_COMBO_ROWS { 1, 1 }
 #define VIAL_UNLOCK_COMBO_COLS { 0, 10 }
+
+#define ENCODERS_PAD_A {GP22, GP4}
```
Comment (ScatteredDrifter) [inline firmware/vial/buran/keymaps/vial/config.h:12]: thanks for adding!

**Review by ScatteredDrifter — APPROVED** [review]
Hey, thank you for contributing and suggesting changes. 

I've lost track of those aspects or was primarily distracted by university, hence only realizing your PR right now, I'm very sorry for this utterly late response! 
Even more so I'm thankful for your contribution and fixing my stupid issues. 

I approve your changes and apologize for my late response

**Pull request merged**